### ISSUE TITLE
Reading bam files from BBmap (merge)

### ISSUE THREAD

**Issue by alexjvr1** [ANGSD/angsd#306]
Hi all, 

I'm trying to read bam files into Angsd that have been merged with BBmap (merge) and then corrected by MapDamage. I have no problems when working with the original bam files (mapped to the ref with  bwa mem). However Angsd seems to filter out all the data when I read the merged bam. 

A similar issue has previously been reported with [MiniMap2 bam files](https://github.com/ANGSD/angsd/issues/260), but I'm not sure if this was ever resolved. 

Does anyone know what the problem could be and how I can correct the bam file? 

```
[bammer_main] 1 samples in 1 input files
-> Parsing 1 number of samples
No data for chromoId=0 chromoname=LR761647.1
This could either indicate that there really is no data for this chromosome
Or it could be problem with this program regSize=0 notDone=0

-> Done reading data waiting for calculations to finish
-> Done waiting for threads
-> Output filenames:
	->"angsdput.arg"
-> Fri Apr 17 16:45:14 2020
-> Arguments and parameters for all analysis are located in .arg file
-> Total number of sites analyzed: 0
-> Number of sites retained after filtering: 0 
[ALL done] cpu-time used =  4.24 sec
[ALL done] walltime used =  4.00 sec
```

Cheers
Alex

**Comment by ANGSD** [comment]
Dear Alex I have not tried using bamdamage as input but i would be very interested in making this work. Could you maybe supply me with one of the bams?

Best

**Comment by alexjvr1** [comment]
Hi, 

Great, thank you for looking into this. I attach three bam files, all one chromosome of the same individual. They're all bam, but I had to add the .txt extension to upload here. 

1. Original mapped bam: AH07_LR761647.1.bam.txt

[AH07_LR761647.1.bam.txt](https://github.com/ANGSD/angsd/files/4528460/AH07_LR761647.1.bam.txt)

2. Merged PE reads using BBmerge: AH07_LR761647.1_bbmerge.bam.txt
[AH07_LR761647.1_bbmerge.bam.txt](https://github.com/ANGSD/angsd/files/4528459/AH07_LR761647.1_bbmerge.bam.txt)

3. MapDamage correction of PHRED scores on merged bam: AH07_LR761647.1_rescaled.bam.txt
[AH07_LR761647.1_rescaled.bam.txt](https://github.com/ANGSD/angsd/files/4528458/AH07_LR761647.1_rescaled.bam.txt)

Please let me know if you need anything more. 
Best
Alex

**Comment by ANGSD** [comment]
Hi Alex, I fixed the issue that you are referencing concerning the minimap bam files. There was indeed an issue with supplemental reads. And that should be fixed in this commit: c9b24d1fc552494

But I dont think this is what is causing the problem alone for these bams. Per default PE reads where they dont map to the same chrs is discarded this is default in both samtools and angsd. You can disable this with -only_proper_pairs 0. But please use the most recent version of angsd since you do have some supplmental reads in your bam. 

I am closing this issue, but feel free to reopen if needed.

Best
